common/activityLog/activityLog.js

```diff
@@ -60,6 +60,7 @@ export const type = {
   processingStepDelete: 'processingStepDelete',
   processingStepCalculationCreate: 'processingStepCalculationCreate',
   processingStepCalculationIndexUpdate: 'processingStepCalculationIndexUpdate',
+  processingStepCalculationUpdate: 'processingStepCalculationUpdate',
```

Comment:
is the activity log in dashboard home show the right content ?

Reply:
no, it doesn't, I'll fix it